src/style.scss

```diff
@@ -1,5 +1,2 @@
-// @import '_component/config/color';
-// @import '_component/config/typography';
-// @import '_component/layout/layout';
-// @import '_component/header/header';
-// @import '_component/footer/footer';
\ No newline at end of file
+h1 {color: blue;
```

Comment:
Although it's OK, I want you to move the color: blue on a new line to increase readability